crates/bevy_ecs/src/system/system_registry.rs

```diff
@@ -164,12 +164,9 @@ impl World {
     ///
     /// ```
     /// # use bevy_ecs::prelude::*;
-    /// #[derive(Resource, Default)]
-    /// struct Counter(u8);
```

Comment:
I think we should keep the `Counter`, as a simple way to show the user that we can use `Resource`s and so. Looking the code below that, we have `PlayerScore`, which maybe is strange to keep at the same time we remove `Counter`.

Reply:
The `Counter` was being used as a `Local`. If we replace it with `Res<Counter>` that actually changes the meaning of what the example is saying (which is that "duplicate" system registrations actually maintain their own state).

Reply:
Fair enough. So there is no problem removing it after all. LGTM